### PR TITLE
Enable showing extra links if there's an error on app pages

### LINKED ISSUE #1093: [FEATURE REQUEST] Redirect if app is region locked
### Current Behavior (if any)
When you visit an app page in the Steam Store, that is region locked, it will give you an error message.

### Desired Behavior
Since just this error message is useless, I propose to add links or redirect to SteamDB and/or the steam community page.

### Example
https://store.steampowered.com/app/239140/Dying_Light/ for german users
image

### Additional Info
https://github.com/tfedor/AugmentedSteam/issues/294#issuecomment-544275854

### PR DESCRIPTION
**Pull request by candela97**
Closes #1093

### CODE REVIEW

File: src/js/Content/Features/Store/Common/FExtraLinks.js [hunk 1]
```diff
@@ -38,128 +45,127 @@ export default class FExtraLinks extends Feature {
...
+        const isAppPage = this._type === "app";
+        const appName = typeof this.context.appName === "string" && HTMLParser.clearSpecialSymbols(this.context.appName);
+
         // Note: Links should be rendered in the same order as displayed on the options page
         const links = [
             {
-                "id": "showitadlinks",
-                "className": "itad_ico",
+                "link": `https://steamcommunity.com/app/${this._gameid}/`,
+                "text": "Community Hub",
```
Comment (MxtOUT) [inline src/js/Content/Features/Store/Common/FExtraLinks.js:55]: Isn't that a localizable string?

**Comment by MxtOUT** [comment]
Also, the position could be a bit nicer

image

File: src/js/Content/Features/Store/Common/FExtraLinks.js (hunk 1 above)
Comment (candela97) [reply src/js/Content/Features/Store/Common/FExtraLinks.js:55]: I think we have "Community", but not "Community Hub", will need a new one

**Comment by candela97** [comment]
> Also, the position could be a bit nicer

Yeah, any ideas? I'm also thinking about moving the "Community Hub" link to roughly where it is on normal app pages.

**Comment by MxtOUT** [comment]
How about centering it?
Pushing it to the right will also make it look misplaced.

**Pull request merged**